support toggle dark mode between light theme or dark theme

taquangtrung opened the pull request:
Hi,

I implemented a small PR to allow users quickly switch between the light theme and dark theme of dark mode.

This is relevant to this PR: https://github.com/emacs-eaf/eaf-browser/pull/43

Can you consider merging it?

Thanks!

manateelazycat commented:
why remove code of update_theme？

it use change color of browser when emacs change theme automatically

taquangtrung commented:
Oh, I didn't see the function `update_theme` is called else where, so I removed that code.

I just reverted it back, but add a condition to control when to reload the configuration from Emacs.

manateelazycat commented:
looks good，thanks
